application/src/main/resources/locale/material/horizons/encoded.csv

```diff
@@ -1,5 +1,9 @@
-messageKey,,en,de,es,fr,pt,ru
 material.encoded.adaptiveencryptors,Adaptive Encryptors Capture,Adaptive Encryptors Capture,Adaptive Verschlüsselungserfassung,Captura de encriptadores adaptativos,Capture de cryptage évolutif,Encriptadores adaptativos capturados,Захват адаптивного шифровальщика
+material.encoded.ancientbiologicaldata,Pattern Alpha Obelisk Data (Guardian),Pattern Alpha Obelisk Data (Guardian),Alpha-Muster-Obeliskendaten,Datos de obelisco de patrón alfa,Données d''obélisque de type alpha,Dados de Obelisco de Padrão Alfa,Данные с обелиска «Альфа»
```

Comment:
keep the order of the file the same. new keys are always added to the end, so keeping it up to date is easier